README.md

```diff
@@ -304,6 +314,13 @@ Reboot again and there should be no alarms.
 
 If you receive any errors or alarms after rebooting, please refer to the [Troubleshooting](#troubleshooting) section for information on how to remedy the issue.
 
+If `SAVE DATA CRC CHECK FUNC (FSU)` was disabled at the start of this procedure, then it can now be reenabled.
+
+ 1. upgrade to *SAFETY* security level by touching `[System Info]`→`[Security]` (default password is all `5`'s)
+ 1. touch `[Setup]`→`[Function Enable]`
+ 1. navigate to `SAVE DATA CRC CHECK FUNC (FSU)`
+ 1. set this feature to `VALID`
```

Comment:
should we perhaps put this above the "If you receive any errors or alarms .." sentence above it?

It's still part of the normal setup procedure, right? Then after this section, we explain where to find troubleshooting information.

Reply:
I like where it is. If they get any errors, then it's likely that the Speed FB didn't execute. In that case, they need to leave the CRC check off while they troubleshoot the other problems.